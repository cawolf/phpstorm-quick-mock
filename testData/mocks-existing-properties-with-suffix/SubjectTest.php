<?php

namespace App\Tests;

use App\SubjectUnderTest;
use App\TestCase;

class SubjectTest extends TestCase
{
    /** @var SubjectUnderTest */
    private $subject;

    public function setUp()
    {
        parent::setUp();
        $this->subject = new SubjectUnderTest<caret>();
    }
}
