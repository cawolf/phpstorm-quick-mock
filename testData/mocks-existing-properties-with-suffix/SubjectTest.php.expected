<?php

namespace App\Tests;

use App\SubjectUnderTest;
use App\TestCase;

class SubjectTest extends TestCase
{
    /** @var string */
    private $dataQuickMocked;
    /** @var SubjectUnderTest */
    private $subject;

    public function setUp()
    {
        parent::setUp();
        $this->dataQuickMocked = '';

        $this->subject = new SubjectUnderTest($this->dataQuickMocked);
    }
}